### DISCUSSION TITLE
arthas trace 怎么和OpenTelemetry里面的trace自动关联起来

### DISCUSSION THREAD

**Discussion by mitht** [alibaba/arthas#2895] [Q&A]
可以将两个trace关联起来嘛,大佬们有没有什么解决方案

**Comment by WangJi92** [comment]
https://github.com/alibaba/arthas/issues/1244

**Reply by mitht** [reply]
使用的是 opentelemetry-java-instrumentation 自动打桩注入的方式,业务代码没有引入 OpenTelemetry的包,两者可以关联起来嘛
